### ISSUE TITLE
Unable to read Disc Issue with PS2 SCHP-90004

### ISSUE THREAD

**Issue by NilKumar123** [CTurt/FreeDVDBoot#131]
Hello, I am getting the "unable to read disc" issue, I have tried a real DVD Movie (One Night at McCool's which says "compatible with ps2 which plays no with no issue), I have tried the 3.10 + 3.11 hybrid iso and I have tried the "All PS2 Slim - English Language.iso". I have also tried the other suggestions from the comments (removing controller, memory cards, etc) but I still get the "Unable to read disc" issue.

My PS2 version information is as follows:
SCHP-90004
Browser1.40
CD Player 2.00
Playstation Driver 2.00
DVD Player 3.11E

Thanks

**Comment by dexcrush** [comment]
I am having the same issue. When I start the console. The black screen flickers and it opens directly inside the connected devices screen. If I open and close the disk flap. It just get stuck at reading disk. I have this same version console.

**Comment by Comix93** [comment]
I have a SCHP-90004 slim ps2.  8D and when I put the DVD-R on I get a black screen and after a few minutes a white writing error but celte times this white writing does not appear and the black screen remains I waited for about 30 minutes and still nothing help me.

**Comment by hansi122435** [comment]
What kind of blank disc did you use?

**Comment by NilKumar123** [comment]
> What kind of blank disc did you use? And: Which burning-program?

I used a few, Sony, Verbatim, Baseline and a non-branded one, I used CDBurnerXP at 1x and compatibility mode with all the necessary setting everyone else used on the forum lastly all the DVD's were DVD-R as other commenters were saying that DVD-RW wasn't working.

**Comment by mheiniger** [comment]
Have you tried reading a normal copy of a movie on a DVD-R ?
I also have a SCHP-90004 ps2 and it almost immediately gives me a disc-error on any DVD-R's, be it FreeDVDBoot or just a movie.
Normal games and dvd's run fine.

**Comment by NilKumar123** [comment]
> Have you tried reading a normal copy of a movie on a DVD-R ?
> I also have a SCHP-90004 ps2 and it almost immediately gives me a disc-error on any DVD-R's, be it FreeDVDBoot or just a movie.
> Normal games and dvd's run fine.

Yes, I tried an original DVD and a burned DVD of which both read and play.
